### DISCUSSION TITLE
How to display all the data?

### DISCUSSION THREAD

**Discussion by shanbeiannan** [microsoft/fluentui#26401] [Q&A]
Hello everyone,
I have a problem, when I display data to DetailList component, it only shows part of the whole. I don't know how I set it up. I saw the demo and it has the same problem. like this:
![image](https://user-images.githubusercontent.com/28774690/213184254-2e249df8-f4c5-44ac-a245-2b4515b104fb.png)

![image](https://user-images.githubusercontent.com/28774690/213184136-8ea56ebb-0dd2-487d-821c-40b592bd80a1.png)

Except the data is not showing in the view, but there is a div, and adding the height just shows no data.

**Comment by shanbeiannan** [comment]
[The Demo](https://developer.microsoft.com/en-us/fluentui#/controls/web/scrollablepane#usage)

**Comment by micahgodbolt** [comment]
It looks like there is a bug when that demo is hosted on our website. Export it to codepen though and it works properly.

https://codepen.io/micahgodbolt/pen/oNMOWgo?editors=0010
